Update CLI readme (#2207)
snyk/snyk: README.md

```bash
brew tap snyk/tap
brew install snyk
```

</details>

<details>
  <summary>Scoop (Windows)</summary>

### Scoop

Install Snyk CLI from our [Snyk bucket](https://github.com/snyk/scoop-snyk) with [Scoop](https://scoop.sh) on Windows:

```
scoop bucket add snyk https://github.com/snyk/scoop-snyk
scoop install snyk
```

</details>

<details>
  <summary>Snyk CLI in a Docker image</summary>

### Snyk CLI in a Docker image

Snyk CLI can also be run from a Docker image. Snyk offers multiple Docker images under [snyk/snyk-cli](https://hub.docker.com/r/snyk/snyk-cli) and [snyk/snyk](https://hub.docker.com/r/snyk/snyk) ([snyk/images on GitHub](https://github.com/snyk/snyk-images) for more details).

These images wrap Snyk CLI and depending on the Tag comes with a relevant tooling for different projects. For example scanning a Gradle project with snyk/snyk-cli:

```bash
docker run -it
    -e "SNYK_TOKEN=<TOKEN>"
    -e "USER_ID=1234"
    -v "<PROJECT_DIRECTORY>:/project"
    -v "/home/user/.gradle:/home/node/.gradle"
  snyk/snyk-cli:gradle-5.4 test --org=my-org-name
```

</details>

## Install as a part of a Snyk CLI integration

> Snyk also offers many integrations into developer tooling. These integrations will install and manage Snyk CLI for you. For example:

Reviewer: ```suggestion
Snyk also offers many integrations into developer tooling. These integrations will install and manage the Snyk CLI for you. For example:
```

- [Snyk Jenkins plugin](https://github.com/jenkinsci/snyk-security-scanner-plugin)
- [CircleCI Orb](https://github.com/snyk/snyk-orb)
- [Azure Pipelines Task](https://github.com/snyk/snyk-azure-pipelines-task)
- [GitHub Actions](https://github.com/snyk/actions)
- [IntelliJ IDE Plugin](https://github.com/snyk/snyk-intellij-plugin)
- [VS Code Extension](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner)
- [Eclipse IDE Extension](https://github.com/snyk/snyk-eclipse-plugin)
- [Maven plugin](https://github.com/snyk/snyk-maven-plugin)
- And many more. See [the Integrations documentation](https://support.snyk.io/hc/en-us/categories/360000598398-Integrations)

<p align="center">
  <a href="https://support.snyk.io/hc/en-us/categories/360000598398-Integrations">
    <img src="help/ide.svg" alt="Snyk CLI IDE integration" width="50%" />
  </a>
</p>

---

# Getting started with Snyk CLI

> Once you installed Snyk CLI, you can verify it's working by running

Reviewer: ```suggestion
Once you installed the Snyk CLI, you can verify it's working by running
```

```bash
snyk --version
```

> ## Authorization

Reviewer: Should we mention additional configuration options (ex using `SNYK_TOKEN`) as well as talk about the `snyk.json` config file?

Author: Added to the CI/CD section

<p align="center">
  <a href="https://snyk.io">
    <img src="help/monitor.svg" alt="Snyk CLI monitor projects" width="70%" />
  </a>
</p>

```
> snyk monitor
Monitoring /project (project-name)...

Explore this snapshot at https://app.snyk.io/org/my-org/project/29361c2c-9005-4692-8df4-88f1c040fa7c/history/e1c994b3-de5d-482b-9281-eab4236c851e

Notifications about newly disclosed issues related to these dependencies will be emailed to you.
```

### Add Snyk to your CI/CD

Snyk is really powerful when you are continuously scanning and monitoring your projects for vulnerabilities.

Use one of [our integrations](#install-as-a-part-of-a-snyk-cli-integration) to stay secure.

## More flags and options to try

Here are some flags that you might find useful:

- `--severity-threshold=low|medium|high|critical`: Only report vulnerabilities of provided level or higher.
- `--json`: Prints results in JSON format.
- `--all-projects`: Auto-detect all projects in working directory

See all the available commands and options by running `--help`:

```bash
snyk --help
# or get help for a specific command like
snyk iac --help
snyk code --help
```

# Getting support

> We recommend reaching to [support@snyk.io](mailto:support@snyk.io) email whenever you need help with Snyk CLI or Snyk in general.

Reviewer: ```suggestion
We recommend reaching out via the [support@snyk.io](mailto:support@snyk.io) email whenever you need help with Snyk CLI or Snyk in general.
```

GitHub Issues on any [Snyk project](https://github.com/snyk) are not actively monitored by Snyk support.

## Contributing

> This project is open source, but we don't encourage outside contributors.

Reviewer: ```suggestion
This project is open source but we don't encourage outside contributors.
```
